app/src/main/res/layout/card_login_prompt.xml

```diff
@@ -1,42 +1,37 @@
-<android.support.v7.widget.CardView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     style="@style/StandardCard">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@drawable/card_selector"
-        android:gravity="center_horizontal"
         android:orientation="vertical"
         android:padding="@dimen/material_default_padding">
 
         <TextView
+            style="@style/CardTitle"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:fontFamily="sans-serif-medium"
-            android:gravity="center_horizontal"
-            android:text="@string/card_login_title"
-            android:textAppearance="?android:attr/textAppearanceMedium"
-            android:textSize="20sp"/>
+            android:text="@string/card_login_title" />
 
         <TextView
+            style="@style/CardBody"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/material_default_padding"
             android:layout_marginTop="@dimen/material_small_padding"
-            android:text="@string/card_login_body"/>
+            android:layout_marginBottom="@dimen/material_small_padding"
+            android:text="@string/card_login_body" />
 
-        <android.support.v7.widget.AppCompatButton
+        <android.support.design.button.MaterialButton
             android:id="@+id/login_button"
+            style="@style/Widget.MaterialComponents.Button.OutlinedButton"
```

Comment:
I'd personally use a ContainedButton, since connecting to TUMonline is quite an important action

Reply:
I’ll give it a try 👍